src/net/sourceforge/kolmafia/utilities/StringUtilities.java

```diff
@@ -921,4 +922,10 @@ public static String stripHtml(String s) {
   public static String capitalize(final String s) {
     return s.substring(0, 1).toUpperCase() + s.substring(1);
   }
+
+  public static String pascalToCamelCase(final String s) {
```

Comment:
This is UPPER_SNAKE_CASE, not PascalCase.

Reply:
Rather... the conversion is UPPER_SNAKE_CASE (or UPPER_CASE_WITH_UNDERSCORES if you prefer Python's naming) to PascalCase.

Reply:
Fixed.